packages/charts/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -605,7 +605,7 @@ export class HorizontalBarChartWithAxisBase
     const { useSingleColor = false } = this.props;
     const bars = this._points.map((point: IHorizontalBarChartWithAxisDataPoint, index: number) => {
       let shouldHighlight = true;
-      if (this._getHighlightedLegend().length > 0) {
+      if (this.state.isLegendHovered || this.state.isLegendSelected) {
```

Comment:
why this change?

Reply:
to make it consistent with the createNumericBars